src/GSoft.Cqrs.Tests/CqrsBuilderExtensionsTests.cs

```diff
@@ -101,6 +109,42 @@ public async Task AddHandler_Can_Register_Command_Handlers_And_Query_Handlers()
         Assert.Equal("rabbles", res3);
     }
 
+    [SuppressMessage("StyleCop.CSharp.OrderingRules", "SA1201:ElementsMustAppearInTheCorrectOrder", Justification = "Getting this close to its usage.")]
```

Comment:
I understand why you put it there, but I would recommend following the rules to ensure a better consistence accross classes.

I feel like it's easier to have those fields that can be reused across multiple methods at the top. It gives better visibility of what's already available.

Reply:
J'ai hésité. Je l'ai mis là pour la proximité et parce que l'utilisation est limité à quelques méthode. Mais tu as raison je vais les déplacer en haut.